.update-changes.cfg

```diff
@@ -1,6 +1,9 @@
 new_commit_msg="Update CHANGES. [skip ci]"
 show_authors=1
 
+# We rename this to avoid a problem on macOS (see #32).
+file_version="plugin/VERSION.dat"
```

Comment:
WDYT about a more descriptive name, e.g., `VERSION.spicy-plugin`?

Reply:
oh, the commit misses a final change: I wanted this to remain inside the top-level directory (because it doesn't work inside plugin/ either).